pyro.py

```diff
@@ -267,7 +267,7 @@ def __init__(self, solver_name, comp_bench=False,
         self.reset_bench_on_fail = reset_bench_on_fail
         self.make_bench = make_bench
 
-    def run_sim(self):
+    def run_sim(self, rtol):
         """
         Evolve entire simulation and benchmark at the end.
         """
```

Comment:
docstring should be "and _compare to_ benchmark..."

Reply:
Fixed in 769a64d